commons/test/src/com/fulcrumgenomics/commons/io/AsyncStreamSinkTest.scala

```diff
@@ -35,7 +35,7 @@ import scala.collection.mutable
   */
 class AsyncStreamSinkTest extends UnitSpec {
   "AsyncStreamSink" should "capture all the output" in {
-    val file = Paths.get("src/test/resources/com/fulcrumgenomics/commons/io/async-stream-sink-test.txt")
+    val file = Paths.get("commons/test/resources/com/fulcrumgenomics/commons/io/async-stream-sink-test.txt")
```

Comment:
I'm a bit surprised by this. Does mill require even for single-module builds that the main and test sources are within a module directory?

Reply:
http://www.lihaoyi.com/mill/page/common-project-layouts.html